.gitignore

```diff
@@ -4,10 +4,6 @@
 
 ### Code ###
 .vscode/*
```

Comment:
![image](https://user-images.githubusercontent.com/16801528/58001456-c9962e80-7adb-11e9-9c36-6cecf34b13af.png)
What changed your opinion on this?

Reply:
found out that the current setting are about the same as the default ==> don't need the in git

Reply:
:)